<ui version="4.0" >
 <class>WindowsDialog</class>
 <widget class="QDialog" name="WindowsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Windows</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QListView" name="windowListView" >
     <property name="whatsThis" >
      <string>A list of the application's currently opened windows</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="activateWindowButton" >
       <property name="whatsThis" >
        <string>Activates the selected window and closes this dialog</string>
       </property>
       <property name="text" >
        <string>Activate</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeWindowButton" >
       <property name="whatsThis" >
        <string>Closes the selected window</string>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="whatsThis" >
        <string>Closes this dialog</string>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>378</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel" >
     <x>213</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
